source/numerics.tex

```diff
@@ -4068,7 +4068,7 @@
    update $\tcode{begin[}k+q\tcode{]}$ by xoring it with $r_4$,
    and
    set $\tcode{begin[}k\tcode{]}$ to $r_4$.
```

Comment:
This could be written a bit more clearly as `set \tcode{begin[$k$]} to $r_4$.` (i.e., don't set the whole `\tcode` bit in math mode). Similarly, a couple lines previous.

Reply:
Thanks, this even catches some missing maths formatting!

@jwakely: I'm stealing your PR.